test/test.js

```diff
@@ -26,70 +25,86 @@ describe('cypress-terminal-report', () => {
         resolve();
       });
     });
   }).timeout(60000);
-
   it('Logs FETCH API routes.', async () => {
     await new Promise(resolve => {
       exec(commandBase() + 'apiRoutes.spec.js', (error, stdout, stderr) => {
         if (stderr) {
           console.error(stderr);
         }
-
         expect(stdout).to.contain('Method: PUT\n');
         expect(stdout).to.contain('Url: https://example.cypress.io/comments/10\n');
-
         // cy.route empty body.
-        expect(stdout).to.contain('cy:route ⛗  Status: 200 (getComment)\n');
+        expect(stdout).to.contain('cy:route ⛗');
+        expect(stdout).to.contain('Status: 200 (getComment)\n');
         expect(stdout).to.contain('Response: EMPTY_BODY\n');
-
         // cy.route text.
-        expect(stdout).to.contain('cy:route ⛗  Status: 403 (putComment)\n');
+        expect(stdout).to.contain('cy:route ⛗');
+        expect(stdout).to.contain('Status: 403 (putComment)\n');
         expect(stdout).to.contain('Response: This is plain text data.\n');
-
         // cy.route unknown.
-        expect(stdout).to.contain('cy:route ⛗  Status: 401 (putComment)\n');
+        expect(stdout).to.contain('cy:route ⛗');
+        expect(stdout).to.contain('Status: 401 (putComment)\n');
         expect(stdout).to.contain('Response: UNKNOWN_BODY\n');
-
         // cy.route logs.
-        expect(stdout).to.contain('cy:route ⛗  Status: 404 (putComment)\n');
+        expect(stdout).to.contain('cy:route ⛗');
+        expect(stdout).to.contain('Status: 404 (putComment)\n');
         expect(stdout).to.contain('Response: {"error":"Test message."}\n');
-
         // log failed command
         expect(stdout).to.contain('cy:command ✘  get\tbreaking-get\n');
+        resolve();
+      });
+    });
+  }).timeout(60000);
+  it('Logs cy.requests', async () => {
+    await new Promise(resolve => {
+      exec(commandBase('printLogs=always') + 'requests.spec.js', (error, stdout, stderr) => {
+        expect(stdout).to.contain(
+          'cy:request ✔  https://jsonplaceholder.cypress.io/todos/1\n\t\t    Status: 200 \n      \t\t    Response: {\n\t\t      "userId": 1,\n\t\t      "id": 1,\n\t\t      "title": "delectus aut autem",\n\t\t      "completed": false\n\t\t    }'
+        );
+        expect(stdout).to.contain(
+          'cy:request ✔  POST https://jsonplaceholder.cypress.io/comments\n\t\t    Status: 201 \n      \t\t    Response: {\n\t\t      "id": 501\n\t\t    }\n\n\n\n\r'
+        );
+        // log failed command
+        expect(stdout).to.contain(
+          'cy:request ✘  PUT https://jsonplaceholder.cypress.io/comments\n\t\t    Status: 404 - Not Found\n      \t\t    Response: {}\n\n\n\n\n\n'
+        );
+
+        expect(stdout).to.contain(
+          'cy:request ✘  GET https://cypress.free.beeceptor.com/response500\n\t\t    Status: 500 - Server Error\n      \t\t    Response: Hey ya! Great to see you here. Btw, nothing is configured for this request path. Create a rule and start building a mock API.\n\n\n\n\r'
```

Comment:
I created a mock endpoint `https://cypress.free.beeceptor.com` because I want to test error response is correctly printed.